internal/network_test_utils.go

```diff
@@ -115,7 +115,7 @@ func (l localNetworkServices) Serve(ctx context.Context) error {
 		for path, contents := range l.webpages {
 			webpage := contents
 			http.HandleFunc("/"+path, func(w http.ResponseWriter, r *http.Request) {
-				fmt.Fprintf(w, webpage)
+				fmt.Print(w, webpage)
```

Comment:
I don't think this is correct, [`Print` writes to stdout](https://golang.org/pkg/fmt/#Print) but we want to write to `w` here

Reply:
`fmt.Fprint` would be the right function

Reply:
Oh, right. Thanks. Why didn't it fail? Let me check.

Reply:
The CNI test failed in buildkite due to this